front/src/common/api/generatedEditoastApi.ts

```diff
@@ -3736,6 +3752,29 @@ export type TrainScheduleBase = {
   start_time: string;
   train_name: string;
 };
+export type PacedTrainBase = TrainScheduleBase & {
+  /** Duration of the mission, an ISO 8601 format is expected */
+  duration: string;
+  /** Time between two trains, an ISO 8601 format is expected */
+  step: string;
+};
```

Comment:
We should use the same name convention for docs
```suggestion
export type PacedTrainBase = TrainScheduleBase & {
  /** Duration of the paced train (or service), an ISO 8601 format is expected */
  duration: string;
  /** Time between two occurrences, an ISO 8601 format is expected */
  step: string;
};
```